README.md

```diff
@@ -1,99 +1,78 @@
-
-`$ createuser postgres`
-
-#### Auth0 Account
-
-Liaise with the team members to get an account on Auth0. Once you have this, log into Auth0 and visit `Applications` to get the `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET` values for the `.env` files (see next section)
-
-#### Yarn
-
-Install Yarn on MacOS with Homebrew:
-
-`$ brew install yarn`
-
-#### Bundler
-
-Install Bundler:
-
-`$ gem install bundler`
+- [Docker](https://docs.docker.com/docker-for-mac) greater than or equal to `18.03.1-ce-mac64 (24245)`
+- Access to the dxw 1Password vault named `DSS` can be granted by anyone on the dxw technical operations team
 
 ### Setting up the project
 
-Copy `.env.development.example` to `.env.development`. This file contains some
-secrets which need to be set. Liaise with team members to get the required
-values. See [environment-variables.md](/docs/environment-variables.md) for a
-description of all these environment variables.
-
-Once this is done, use Bundler to set up the project:
-
-`$ bundle install`
-
-Create & set up the database:
+Copy the example environment variable file used with Docker:
 
-`$ bundle exec rake db:setup`
+```bash
+$ cp docker-compose.env.sample docker-compose.env
+```
 
-Next, copy `.env.test.example` to `.env.test`. This file needs a `SECRET_KEY_BASE` parameter which you can generate with Rake:
+From the 1Password vault, copy the contents of a file named `Docker Compose - API - docker-compose.env` into the new `docker-compose.env` file. This file should contain no further additions to get started.
 
-`$ rake secret`
+The most common command used to start all containers. It does database set up if required and leaves you on an interactive prompt within the rails server process where `pry` can be used for debugging:
 
-Install front-end dependencies:
+```bash
+$ bin/dstart
+```
 
-`$ yarn install`
+If you'd like to see all logs, like `Sidekiq` or `Redis` you can use the conventional docker-compose command - you will lose the ability to use `pry`:
 
-### Seeding the database
+```bash
+$ docker-compose up
+```
 
-`db/seeds.rb` is out of date at the time of writing. Instead, ask the team for a dump of the live database.
+If you'd like to shut all containers down, and remove database information persisted in `docker volumes` you can run the following command which rebuilds everything from scratch:
 
-Once you have the database dump, you can import it into your local Postgres with:
+```bash
+$ bin/drebuild
+```
 
-`$ pg_restore -h localhost -U postgres -d DataSubmissionServiceApi_development /path/to/database/dump.tar`
+---
 
 ## Running the tests
 
-To run the rspec tests:
+Because the setup and teardown introduces quite some latency, we use the spring service to
+start up all dependencies in a docker container. This makes the test run faster.
+
+Get the test server up and running behind the scenes:
 
-`$ bundle exec rspec`
+```bash
+$ bin/dtest-server up
+$ bin/dtest-server down
```

Comment:
I think that if I saw this, I would (stupidly, possibly) assume that I was meant to run `up` and then `down`. Should we just put the `up` command here?

Reply:
Yes I think that's entirely reasonable and something we should try to avoid!